evasquez26/angular4-startup
# S5-ONE-213 Accept/reject request to join a course as teacher

## Story / Task

see [ONE-213](https://bixlabs.atlassian.net/browse/ONE-213)

## What this PR does?

Updates ticket management section to accept/reject a role request for join a course as teacher

### How to test / reproduce

- Login to the app with a user with admin privileges inside an organization
- Go to admin (ticket management)
- See there's a list of role requests
- Find the request for course teacher
- See you can go to details
- Accept or reject the request
- See the request is marked as closed

### TODOs
- [x] Rebase after merging #48 

### Related PRs
N/A

## Migrations
 NO


Thanks
